package.json

```diff
@@ -46,7 +64,7 @@
     "test:server": "jest --config ./jest.server.js",
     "test:report": "yarn test --ci --silent || true",
     "test:samples": "yarn workspace @okta/test.e2e.samples start",
-    "test:integration": "jest --config ./jest.integration.js",
+    "test:integration": "jest --runInBand --config ./jest.integration.js",
```

Comment:
is there a reason these tests cannot run in parallel?

Reply:
myaccount api need to be tested in a crud flow, run it in parallel causes flakiness.